src/cloud/paths/tasks.yml

```diff
@@ -147,14 +147,51 @@ post:
     Creates a [task]({{% INFLUXDB_DOCS_URL %}}/process-data/) and returns the created task.
 
     Use this endpoint to create a scheduled task that runs a Flux script.
-    In your task, provide one of the following:
-
-      - _`flux`_ property with an inline Flux script
-      - _`scriptID`_ property with an [invokable script](#tag/Invokable-Scripts) ID
 
     #### InfluxDB Cloud
 
-      - Requires either _`flux`_ (Flux query) or _`scriptID`_ (invokable script ID) in the task.
+    - You can use either `flux` or `scriptID` to provide the task script.
```

Comment:
```suggestion
    - You can use either use `flux` or `scriptID` to provide the task script.
```

Reply:
> - Requires either...

I'm trying to keep the platform/API the subject here as much as possible, e.g. "InfluxDB Cloud... requires...." Doesn't always work, though.

Reply:
sorry i think i read that sentence wrong, please disregard the comment

Reply:
> sorry i think i read that sentence wrong, please disregard the comment

Well, I read the diff and your suggestion wrong when I made the comment.